src/rmutil/vector.c

```diff
@@ -32,6 +32,10 @@ inline int Vector_Pop(Vector *v, void *ptr) {
   return 0;
 }
 
+void* Vector_Data(const Vector *v) {
+  return (void*)v->data;
+}
```

Comment:
We can sort records without making additional allocations by modifying the Vector contents, so I added this interface.